app/org/sagebionetworks/bridge/BridgeUtils.java

```diff
@@ -68,6 +69,16 @@ public static String resolveTemplate(String template, Map<String,String> values)
         }
         return template;
     }
+
+    public static void addWarningMessage(String msg) {
+        Http.Response response = Http.Context.current().response();
```

Comment:
I really dislike accessing this thread local variable in a static utility class like this... I would make this method a base method in BaseController, which is part of the Play infrastructure. That's because this code will have to be rewritten if we switch off of the Play Framework.